modules/master-node-sizing.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-host-practices.adoc
+
+[id="master-node-sizing_{context}"]
+=  Master node sizing
+
+Control plane density focused tests were run at different node scale to determine the master node resource requirements depending on the number of nodes in the cluster. The recommended sizes are
+----
+25 nodes: 4 CPU cores and 16GB memory
+100 nodes: 8 CPU cores and 32GB memory
+250 nodes: 16 CPU cores and 64GB memory
+----
+
+====
+[NOTE]
+In {product-title} {product-version}, half of a CPU core (500 millicore) is now reserved by the system by default compared to {product-title} 3.11 and previous versions. The sizes are determined taking that into consideration.
```

Comment:
1 core is reserved by the system in the case of OpenShift dedicated like @jeremyeder mentioned, should we mention that here or just in the OpenShift dedicated docs?